server/models/GameLogic.js

```diff
@@ -350,9 +350,7 @@ class GameLogic {
         this.newCard();
         this.update({ status: statusTypes.DISPLAY_ALL_VOTES });
         this.calcScores();        
-        if (this.state.mode === 'original') {
```

Comment:
Did you really want to remove this if statement?

Reply:
so this if statement was because sisi was trying to avoid calling the update labels in the custom mode, because that wasn't set up yet, so she just wanted to skip the update if that makes sense. but now that i've made the update labels work for custom mode we don't need the if statement

Reply:
Ah, cool 👍